### PR TITLE
fix(cli): keep narrow child kill hint visible

### LINKED ISSUE #10096: CLI narrow session chooser hides the selected child kill action
## Problem

When the session chooser owns keyboard input at 70 columns, selecting a running child enables the k key, but the footer does not advertise that action:

```text
↑/↓ select · Enter focus · Tab input · Esc input · Ctrl-C stop
```

Pressing k still stops the selected child. The visible controls therefore disagree with the input surface precisely for a destructive action.

## Reproduction

Tested from a source build at origin/main 27188eac0c with TERM=xterm-256color, a 70x22 PTY, and --no-color.

1. Run texra orchestrate --approval-policy ask --no-color in a 70x22 terminal.
2. Start a team workflow that delegates to a research child.
3. Press Tab to open the session chooser.
4. Press Down to select the running research child.
5. Observe that the footer omits k kill.
6. Press k; the research child is stopped.

The parent continued correctly and explicitly reported that the cancelled independent check was unavailable. The defect is confined to the chooser affordance.

## Cause

The full child-list footer with k kill occupies 71 cells, so it does not fit inside the 70-column terminal chrome. The next fitting candidate removes k kill but retains Enter focus. A narrower ordering can retain the action that is specific to the selected running row while omitting the less urgent focus hint.

## Expected behavior

When the selected row is killable and the terminal has room for a compact hint, the chooser footer should continue to show k kill. The visible footer and accepted keys should agree.

## Bounded acceptance criteria

- At 70 columns, a killable selection advertises k kill together with selection, input-return, and stop/exit recovery controls.
- A non-killable selection does not advertise k kill.
- Wide-terminal wording and key behavior remain unchanged.
- A focused status-bar test covers the narrow fitting order.

### PR DESCRIPTION
**Pull request by LionSR**
Closes #10096.

## Summary

- keep `k kill` visible for a selected running child in a narrow session chooser
- retain `Enter focus` when the selected row is not killable
- remove a dominated status-bar candidate that could never win

## Ownership decision

`statusBarDisplay` remains the single authority for fitting child-list controls. The selected row contributes only the fact that killing is currently available. The fitter then prefers the row-specific `k kill` action over `Enter focus` when necessary, but never drops `Enter focus` for a non-killable selection. Key dispatch and child lifecycle ownership are unchanged.

## Validation

- StatusBar suite: 97 tests passed
- workspace typecheck
- focused lint
- prior full CI and PTY validation were green before the rebase

### CODE REVIEW

File: packages/cli/src/chat/tui/panes/statusBarDisplay.ts [hunk 1]
```diff
@@ -801,6 +801,13 @@ function childListBindingsText(
         escBinding,
         ctrlCBinding,
       ]),
+      statusBarBindingRow([
+        selectBinding,
+        killBinding,
+        tabBinding,
+        escBinding,
+        ctrlCBinding,
+      ]),
```
[the remark this answers is not in the document]
Comment (LionSR) [reply packages/cli/src/chat/tui/panes/statusBarDisplay.ts]: Fixed in ce9a4d472c. The compact candidate is now gated on `selectionKillable`, and a 55-column non-killable regression test preserves `Enter focus · Esc input · Ctrl-C exit`. The focused status suite passes all 96 tests.

**Comment by LionSR** [comment]
Addressed the review edge case in ce9a4d472c: the compact row now exists only for a killable selection, and a new 55-column non-killable test verifies that `Enter focus` is retained. Focused status suite: 96/96 passed.

**Comment by LionSR** [comment]
Also addressed the dominated-candidate review in 511e1db15e. The old `select + Enter + Tab + Esc` row was identical to an earlier candidate for non-killable selections and wider than the compact kill-preserving row for killable selections, so it is deleted. Focused status suite remains 96/96.

File: packages/cli/src/chat/tui/panes/statusBarDisplay.ts [hunk 2]
```diff
@@ -801,6 +801,13 @@ function childListBindingsText(
...
       statusBarBindingRow([
```
[the remark this answers is not in the document]
Comment (LionSR) [reply packages/cli/src/chat/tui/panes/statusBarDisplay.ts]: Fixed in 511e1db15e: the dominated candidate was deleted. The focused status suite passes 96/96.

File: packages/cli/src/chat/tui/panes/statusBarDisplay.ts [hunk 3]
```diff
@@ -801,6 +801,13 @@ function childListBindingsText(
         escBinding,
         ctrlCBinding,
       ]),
+      statusBarBindingRow([
```
[the remark this answers is not in the document]
Comment (LionSR) [reply packages/cli/src/chat/tui/panes/statusBarDisplay.ts]: Fixed in ce9a4d472c: the compact row is gated on `selectionKillable`, so non-killable selections retain the prior fitting order.

File: src/test-kernel/cli/StatusBar.vitest.ts [hunk 4]
```diff
@@ -389,6 +389,22 @@ describe('CLI StatusBar display model', () => {
     expect(display.bindings).not.toContain('Tab sessions');
   });
 
+  it('keeps the selected child kill action at narrow widths', () => {
```
[the remark this answers is not in the document]
Comment (LionSR) [reply src/test-kernel/cli/StatusBar.vitest.ts]: Covered in ce9a4d472c with a 55-column non-killable assertion expecting `Enter focus · Esc input · Ctrl-C exit`.

File: packages/cli/src/chat/tui/panes/statusBarDisplay.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (LionSR) [reply packages/cli/src/chat/tui/panes/statusBarDisplay.ts]: Fixed in ce9a4d472c by gating this candidate on `selectionKillable`.

File: src/test-kernel/cli/StatusBar.vitest.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (LionSR) [reply src/test-kernel/cli/StatusBar.vitest.ts]: Added in ce9a4d472c. The non-killable 55-column regression case preserves `Enter focus · Esc input · Ctrl-C exit`.

**Comment by LionSR** [comment]
Rebased onto current main as the two remaining cascade corrections at 008b1d1f91 and 2ce5aa8b7d. The initial kill-hint feature already landed elsewhere.

@codex review

@texra-ai re-review
